app/Libraries/HandlesProblemBeatmapsetDiscussionPost.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+declare(strict_types=1);
+
+namespace App\Libraries;
+
+use App\Jobs\Notifications;
+use App\Models\BeatmapDiscussion;
+use App\Models\User;
+
+// TODO: should go into namespace
```

Comment:
uh, yeah, how about now instead. Also for all other BeatmapsetDiscussion libraries (`Libraries/BeatmapsetDiscussion`, `Libraries/BeatmapsetDiscussion/Traits`)